### PR TITLE
Add sublabel for Onscreen, Driver, Help

### PR DESCRIPTION
**Pull request by lasers**
What do you think about using `Notifications` instead of `Onscreen Notification` ?

I don't like `Onscreen Overlay` staying at `Onscreen Display`'s house.

If we do change `Onscreen Notification` to `Notification`, then we could put everything together on one menu because the `Notifications` options should stand out easily at bottom.

It's either that or kick `Onscreen Overlay` out of the house. (Imo).

### CODE REVIEW

**Comment by lasers** [comment]
I think I'm done for now.

File: intl/msg_hash_nl.c [hunk 1]
```diff
@@ -266,15 +266,15 @@ const char *msg_hash_to_str_nl(enum msg_hash_enums msg)
       case MENU_ENUM_LABEL_VALUE_FRONTEND_COUNTERS:
          return "Frontend Prestatie Tellers";
       case MENU_ENUM_LABEL_VALUE_HELP:
-         return "Help";
+         return "RetroArch Tour";
```
Comment (RobLoach) [inline intl/msg_hash_nl.c]: Not sure about having "RetroArch" in the description itself. If you're using Lakka, this entry won't apply. Mind just having "Tour"?

Comment (Kivutar) [inline intl/msg_hash_nl.c]: I agree

Comment (lasers) [inline intl/msg_hash_nl.c]: Won't that be a good thing?

>Lakka is the official Linux distribution of RetroArch and the libretro ecosystem.
>
>Each game system is implemented as a libretro core, while the frontend RetroArch takes care of inputs and display. This clear separation ensures modularity and centralized configuration.

Got anything else (longer) than "Tour" in mind? 
```
Information
Configurations
Tour
Quit RetroArch
```

Comment (lasers) [inline intl/msg_hash_nl.c]: Can everybody skip this single commit on Github when merging? If not, I'll go ahead and fix it.

Comment (Alcaro) [inline intl/msg_hash_nl.c]: Not possible. Someone has to either revert or remove+rebase.

**Pull request merged**